src/components/page/HeaderBreadcrumb/helpers/index.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { BreadcrumbItem } from 'reactstrap';
+import { BreadCrumbProps, Page } from '../HeaderBreadcrumb';
+
+const LinkList = (props: Partial<BreadCrumbProps>) => {
+  const { pages } = props;
+
+  const linkList = (pages as Page[]).map((page, key) => {
+    // render breadcrumb items with urls as links or without urls as text nodes
+    let breadCrumbItem: string | JSX.Element;
+    if (page.url && page.url.trim()) {
+      breadCrumbItem = (
+        <Link to={page.url} key={key}>
+          {page.label}
+        </Link>
+      );
+    } else {
+      breadCrumbItem = page.label;
+    }
+    return <BreadcrumbItem key={key}>{breadCrumbItem}</BreadcrumbItem>;
```

Comment:
To reduce the possibility of key's collision, the key string should be ideally namespaced

Reply:
addressed https://github.com/onaio/reveal-frontend/pull/1194/commits/b47e4a81a9db95b07f166b4ea7e015f108edd882